apps/modernization-ui/src/apps/page-builder/components/Subsection/Subsection.tsx

```diff
@@ -24,7 +25,7 @@ export const SubsectionComponent = ({ subsection }: { subsection: PageSubSection
                             <Icon.Edit size={3} /> Edit Subsection
                         </Button>
                         <Button type="button" onClick={() => console.log('BLAH')}>
-                            <img src="/group.svg" /> Group Subsection
+                            <IconComponent name={'group'} size={'s'} /> Group Subsection
```

Comment:
My design presumes the dev will likely start with the svg name, then import it to the component, then just pass the svg name as a prop to the IconComponent.
Unless we get on top of cataloguing all these new components similar to uswds system, this svg icons/ folder might be our one source of truth.
This is just a stopgap, I'm up to changing our strategy, but I'm seeing these new svgs injected in all sorts of weird ways and hoping to funnel all this activity into a component to reduce sprawl. Also this can live alongside the uswds component system for now